src/name_bazaar/ui/components/offering/general_info.cljs

```diff
@@ -9,25 +9,29 @@
     [reagent.core :as r]))
 
 (defn offering-original-owner-line [{:keys [:offering/original-owner :offering/address]}]
-  [:div.ellipsis
-   "Offered by"
-   (when @(subscribe [:offering/active-address-original-owner? address]) " (You)")
-   ": "
-   [:a
-    {:href (path-for :route.user/offerings {:user/address original-owner})}
-    original-owner]])
-
-(defn offering-new-owner-line [{:keys [:offering/new-owner :offering/address]}]
-  (let [active-address-new-owner? @(subscribe [:offering/active-address-new-owner? address])]
+  (dispatch [:public-resolver.record-hash/load original-owner])
```

Comment:
Please don't use this pattern for initial loading as this doesn't necessarily need to work when users goes from one offering to another, since component doesn't need to be remounted. Put logic for initial loading into events. In this case it'd need to go into 2 events: `:offerings.list-item/expanded`, and `:active-page-changed` for `:route.offerings/detail`